CONTRIBUTING.rst

```diff
@@ -14,6 +14,9 @@ Also, please, follow the code style proposed by `PEP8
 <https://www.python.org/dev/peps/pep-0008/>`_. There is a tool called
 pycodestyle, formerly known as pep8, that checks if your code is compliant.
 
+For more details, please refer to `here
+<https://github.com/open-power-host-os/versions>`_.
```

Comment:
My understanding is that the `Contributing code` section refers to contributing with the Python scripts used in the builds.
I'd create a new section `Contributing packages` and write something like:
For details on adding new packages to the project, refer to the `versions repository documentation <link to specific documentation>`_